### DISCUSSION TITLE
use_fates_ed_prescribed_phys = .true.

### DISCUSSION THREAD

**Discussion by hannah0304** [NGEET/fates#1049] [Q&A] [answered]
Hello,
I wonder about the alterations the model undergoes by turning on 'use_fates_ed_prescribed_phys' option.
In the source code, I found that by turning it on, the model will use 'prescribed physiology' but I am not sure what exactly it means.

After examining outputs with and without this option, I have observed a notable difference in Leaf Area Index (LAI). I would appreciate assistance in understanding the factors that trigger this discrepancy.

**Comment by rgknox** [comment]
When prescribed physiology mode is on, the photosynthesis and respiration modules are turned off. Instead of estimating GPP and respiration via biophysics, we apply a simple conceptual model where the user specifies NPP per unit crown area.  See this snippet of code, where this rate of NPP is applied:

https://github.com/NGEET/fates/blob/sci.1.66.0_api.25.5.0/main/EDMainMod.F90#L494-L508

**Comment by ckoven** [comment, marked as answer]
Thanks @hannah0304 for bringing this up. This comments highlights the need for us to add a section that describes each of the reduced complexity configurations in FATES into the FATES tech note. This mode that you ask about is one of them, and is meant to be used specifically for experiments looking at dynamic ecosystem structure, conditional on a set of known growth and mortality rates. Adding this documentation has been on our to-do list for a long time now, apologies (the issue https://github.com/NGEET/fates-docs/issues/8 was opened ~5 years ago). Will try to prioritize adding this documentation soon.

**Reply by hannah0304** [reply]
Thank you all for your answers!
I would really appreciate any updates to the documentation.

**Reply by ckoven** [reply]
Hi @hannah0304 I added a first draft of the descriptions of the reduced complexity configurations to the tech note, see here: https://fates-users-guide.readthedocs.io/projects/tech-doc/en/latest/fates_tech_note.html#fates-reduced-complexity-configurations. Hopefully it helps explain these modes better-

**Reply by hannah0304** [reply]
Awesome! Just checked the document 👍 
Thank you so much for the update.
